### DISCUSSION TITLE
Replace files with duplicate found + rename files

### DISCUSSION THREAD

**Discussion by Alister-Zeus** [qarmin/czkawka#1129] [Ideas]
Say I have a folder with images and I get new images, so I do a search for duplicates

Now I find a duplicate in the new files folder, but it is of higher _quality_ than original, I can move the file into the folder except it has the same name so won't move (and I can't rename either file as a workaround)

**Comment by Falloutman** [comment]
I came here to ask if this feature could be implemented.
